### ISSUE TITLE
Inquiry about datesets

### ISSUE THREAD

**Issue by Unafyx** [cvvsu/RMEP#2]
Hello, could you please share the RGBHSI dataset and the series of datasets that were resampled later, I would like to reproduce this code to learn. Thank you very much!

**Comment by cvvsu** [comment]
Hi,

Thank you for your interest.
I am afraid I do not have the authority to release the RGBHSI dataset. But the results can still be reproduced.

One of the main conclusions is that the spatial and spectral resolutions of images are not highly bounded to the retrieval accuracy.
Thus, you can collect some images in a weather stations by leveraging your smartphone, and retain the model. You can re sample the RGB images and retrain the model again (maybe one model is enough or just fine-tune the pretrained model. ) Once the retrieval accuracy is high enough, then the results are reproduced.


Hope it helps😀

Peifeng
________________________________
